scripts/shared/clusters.sh

```diff
@@ -25,6 +24,8 @@ source ${SCRIPTS_DIR}/lib/utils
 source "${SCRIPTS_DIR}/lib/cluster_settings"
 [[ -z "${cluster_settings}" ]] || source ${cluster_settings}
 
+echo "Running with: clusters="${clusters[*]}", k8s_version=${version}, globalnet=${globalnet}, registry_inmemory=${registry_inmemory}, cluster_settings=${cluster_settings}"
```

Comment:
I don't think you should move it since you'll be conflating flags with other settings.

Instead I recommend you simply print here the clusters (and perhaps other settings that might be relevant for clusters.sh) as it would be nice to see the final values we'll be working with.

Reply:
I think the `echo` of the flags should remain where it was with the values it had.

Here you can add another `echo` which outputs the combined settings of the clusters after it processed cluster_settings.

Reply:
So duplicate the echo with "clusters=" added? Not sure I see the purpose of that. Unless it's possible the flags outputted earlier might change after _cluster_settings_ is sourced? Even so I don't see the purpose of outputting  an intermediate value.

Reply:
Done